buildkite/shared-build-env.dockerfile

```diff
@@ -2,10 +2,10 @@ FROM mrcide/node-docker:master
 
 RUN apt-get install wget
 
-RUN wget -qO - https://adoptopenjdk.jfrog.io/adoptopenjdk/api/gpg/key/public | apt-key add -
-RUN add-apt-repository --yes https://adoptopenjdk.jfrog.io/adoptopenjdk/deb/
+RUN wget -qO - https://packages.adoptium.net/artifactory/api/gpg/key/public | apt-key add -
+RUN add-apt-repository --yes https://packages.adoptium.net/artifactory/deb/
 RUN apt-get update
-RUN apt-get install adoptopenjdk-11-hotspot -y
+RUN apt-get install temurin-11-jdk -y
```

Comment:
For context, adoptopenjdk-11-hotspot was moved, probably renamed, so Mantra updated this line to point to a new location.